test/db_test.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+/*
```

Comment:
something tells me you did not intend to commit this

Reply:
yep, it's in there because we still need to think about how we want to test it, but it is currently not implemented you're right

Reply:
Sounds good, then please add a comment to these files about their status.